### PR TITLE
Assignment 3

### PR DESCRIPTION
**Pull request by mayankrajendrat**

### CODE REVIEW

**Comment by sailesh-ramayanam** [comment]
Why does this PR have HTML Assignment-1.docx, HTML Assignment-2.docx and other files?

**Comment by mayankrajendrat** [comment]
> Why does this PR have HTML Assignment-1.docx, HTML Assignment-2.docx and other files?

Resolved. Updated PR with "Origin/Master"

File: build/config/paths.js [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 		buildDir: "build",
 		clientDir: "src",
-		clientEntryPoint: "src/toggle.js",
+		clientEntryPoint: "src/index.js",
```
Comment (sailesh-ramayanam) [inline build/config/paths.js:19]: You are changing the entry point and also adding a new file `build.jakefile.js`. I hope the previous assignments work seamlessly with this new setup. Please confirm that you checked Assignments 1, 2 and 3 with this new setup.

Comment (mayankrajendrat) [reply build/config/paths.js:19]: We added a more precise entry point for build and all previous assignments will work as expected.

**Review by sailesh-ramayanam — APPROVED** [review]

**Pull request merged**